src/common/infra/config.rs

```diff
@@ -286,6 +289,9 @@ pub struct Common {
     pub usage_org: String,
     #[env_config(name = "ZO_USAGE_BATCH_SIZE", default = 2000)]
     pub usage_batch_size: usize,
+    #[env_config(name = "ZO_DEFAULT_SCRAPE_INTERVAL", default = 15)]
+    // Default scrape_interval value 15s
+    pub default_scrape_interval: u32,
 }
```

Comment:
We need to check the value, maybe user sets it `0` or `1`, it shouldn't less than `5` seconds.